ui-app/client/src/components/PageEditor/functions/PageOperations.ts

```diff
@@ -566,6 +719,17 @@ export class PageOperations {
 			false,
 		);
 
```

Comment:
send false for the deleteExisting parameter. And make the changes in the function to handle the missing definition objects.